yaksh/templates/yaksh/monitor.html

```diff
@@ -98,11 +111,16 @@
         <td> {{ paper.user.username }} </td>
         <td> {{ paper.user.profile.roll_number }} </td>
         <td> {{ paper.user.profile.institute }} </td>
-        <td> {{ paper.get_answered_str }} </td>
+        <td> {% for que in paper.get_attempted_questions %}
```

Comment:
There are two ways to go about it -
1. Please order all the attempted questions. For a given student, the moderator wouldn't know which answer is the latest. Make sure the is some visual aid pointing the moderator towards the latest answer.

2. Or maybe this entire column could just be called **latest attempt** and show only the latest answer from the user and not everything the student has attempted.